examples/applications/simulated-camera/src/communication.rs

```diff
@@ -4,19 +4,18 @@
 use std::sync::Arc;
 use std::{env::args, net::SocketAddr};
 
-use futures::future::join_all;
-use tokio::spawn;
-use tonic::transport::Server;
-use url::Url;
-
 use chariott_common::{
     error::{Error, ResultExt as _},
     shutdown::{ctrl_c_cancellation, RouterExt as _},
 };
-use examples_common::chariott::proto::{
+use chariott_proto::{
     provider::provider_service_server::ProviderServiceServer,
     streaming::channel_service_server::ChannelServiceServer,
 };
+use futures::future::join_all;
```

Comment:
Intentional, yes, but not necessarily required.

Since we used the convention to have all usings (except `crate`, `super` and `std`) in a single block most frequently, I did the same here. I was already adding a new `use` statement (`use chariott_proto::`), hence I would have needed to decide in which of the two blocks to put it. Instead, I merged the two blocks, which made it clear where to put the new `use` statement.

Reply:
I don't think we did this consistently, but whenever possible, I've tried to go with the grouping (separate blocks): imports from `std`, imports from crates and then imports from `crate`. I believe `rustfmt` preserves groups.

We can address the import order consistency in another PR at another time. Meanwhile, I was just wondering if this unrelated change was for a reason.